Frontend/Gmfctn/src/app/modules/+badges/components/full-list-of-achievements/full-list-of-achievements.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+
+import { Achievement } from 'src/app/shared/models/achievement';
+
+@Component({
+  selector: 'app-full-list-of-achievements',
+  templateUrl: './full-list-of-achievements.component.html',
+  styleUrls: ['./full-list-of-achievements.component.scss']
+})
+export class FullListOfAchievementsComponent implements OnInit {
+  achList: Achievement[] = [{icon: '../../../../assets/phoenix.png',
```

Comment:
Opening an closing object should be on a separate lines like this:
`
{
  icon: '../../../../assets/phoenix.png',
  name: 'Exoft turbo power ',
  description: '',
  xp: 5,
  time: new Date('December 17, 2005 03:24:00')
}
`

Reply:
Fixed